inference-engine/tests/functional/inference_engine/ngraph_reader/region_yolo_tests.cpp

```diff
@@ -0,0 +1,150 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+#include "ngraph_reader_tests.hpp"
+
+TEST_F(NGraphReaderTests, ReadRegionYoloNetwork) {
```

Comment:
New test case shall be added with following attributes: do_softmax=false and proper mask value.